test/parallel/test-http2-respond-file-errors.js

```diff
@@ -28,7 +28,7 @@ const types = {
   symbol: Symbol('test')
 };
 
-const fname = path.resolve(common.fixturesDir, 'elipses.txt');
+const fname = path.resolve(common.fixtures, 'elipses.txt');
```

Comment:
If this is one of the code-and-learn tasks I am sorry we probably were not being very clear with the instructions. `common.fixtures` does not actually exist, I believe this should be changed to something equivalent to `require('../common/fixtures').path('elipses.txt')`.